Makefile

```diff
@@ -15,9 +15,10 @@
 # Image repo/tag to use all building/pushing image targets
 DOCKER_REGISTRY ?= ghcr.io/kubestellar/kubestellar
 CMD_NAME ?= controller-manager
+TRANSPORT_CMD_NAME ?= ocm-transport-controller
 IMAGE_TAG ?= $(shell git rev-parse --short HEAD)
-IMAGE_REPO ?= ${DOCKER_REGISTRY}/${CMD_NAME}
-IMG ?= ${IMAGE_REPO}:${IMAGE_TAG}
+IMAGE ?= ${DOCKER_REGISTRY}/${CMD_NAME}:${IMAGE_TAG}
```

Comment:
Now that this Makefile is building two images, let's use distinctive names for each. If there is TRANSPORT_IMAGE, maybe also CTLMGR_IMAGE? Same for makefile target names.

Reply:
@MikeSpreitzer yes, I agree.
in this PR I tried to avoid it cause there are scripts that uses make commands with the existing names. 
let's do in a followup PR.